cognite/transformations_cli/commands/deploy/transformations_api.py

```diff
@@ -251,8 +255,12 @@ def upsert_notifications(
                 {e.id: (ext_id, e.destination) for e in existing_notif if e.destination not in requested_destinations}
             )
             to_create += [e for e in requested_notif if e.destination not in existing_destinations]
-        client.transformations.notifications.delete(list(to_delete.keys()))
-        client.transformations.notifications.create(to_create)
+        delete_external_ids = list(to_delete.keys())
```

Comment:
`to_delete` instead? Or `to_delete_external_ids` if that is important.

Reply:
Seems to me to be a case of converting a dict to a list. At least using the same name, although allowed, is not best practice as you mess with the type checker

Reply:
I agree that `to_delete_external_ids` is a bit better though

Reply:
to_delete is used for another list and it will be referenced again later, I'll to to_delete_external_ids